admin-client/package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "date-fns": "^2.15.0",
+    "js-file-download": "^0.4.11",
```

Comment:
Instead of pulling this in as a dependency, is this simple enough to write ourselves?

Reply:
@drewbo is confident that new browser APIs make this approach unnecessary, but uncertain whether we still have to worry about the Safari issue noted in the code I linked immediately above.

Reply:
It looks like all the [relevant browser versions handle the download attribute](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/download#browser_compatibility). Do you want to try out writing the functionality so we don't need the added dependency?

Reply:
I'm already in the process of doing that for this PR. 

But note that `js-file-download` is also required in the React app where it's an existing dependency of the `downloadBuildLogsButton` component. If we'd like to do away with it completely I think that's another issue (which I'd be glad to tackle).

Reply:
fd0d58d removes this PR's new admin client dependency on `js-file-download` (and has been tested successfully in Safari 16.5/Mac as well as in Chrome).

Reply:
Awesome.